src/main/java/com/google/graphgeckos/dashboard/components/Log.java

```diff
@@ -31,11 +31,16 @@
   /**
    * Constructs an instance of Log. Both type and link can be null.
    */
-  Log(String type, String link) {
+  public Log(String type, String link) {
```

Comment:
Why do we need two `Log` constructors? Wouldn't one suffice?

Reply:
Because one of them is used in your tests and another is used for parsing

Reply:
I'll change that in a future PR